tests/frame/test_common.py

```diff
@@ -312,6 +324,30 @@ def test_join(df_raw: Any) -> None:
     compare_dicts(result_native, expected)
 
 
+@pytest.mark.parametrize("constructor", [pl.DataFrame, pd.DataFrame])
+@pytest.mark.filterwarnings("ignore:the default coalesce behavior")
+def test_left_join(constructor: Any) -> None:
+    data_left = {"a": [1, 2, 3], "b": [4, 5, 6]}
+    data_right = {"a": [1, 2, 3], "c": [4, 5, 6]}
```

Comment:
can we add another test case where there's yet another overlapping column? like, add `'d': [1,4,2]` to both of them

Reply:
Alright.